commands/features.go

```diff
@@ -0,0 +1,54 @@
+package commands
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/ConradIrwin/font/sfnt"
+)
+
+// Features prints the gpos/gsub tables (contains font features).
+func Features() {
+	if len(os.Args) < 2 {
+		panic(fmt.Errorf("Specify a font file"))
```

Comment:
Can we start to improve the code for handling these errors, we should make it clear how to fix the problem, and make the error message more legible. (I appreciate the other commands aren't yet very good).

```
fmt.Printf("Usage: font features <font-file>")
os.Exit(1)
```

Reply:
I'll do that in a later PR.